.gitignore

```diff
@@ -65,6 +65,9 @@
 /fbw-common/msfs-avionics-mirror/src/sdk/build
 /fbw-common/msfs-avionics-mirror/src/sdk/build/**
 
+!/fbw-msfs-checklist-fix/**
+/fbw-msfs-checklist-fix/out
```

Comment:
Can you add - this makes sure the structure within these folders is identical:

/fbw-msfs-checklist-fix/*

!/fbw-msfs-checklist-fix/README.md

!/fbw-msfs-checklist-fix/docs/

!/fbw-msfs-checklist-fix/docs/**

!/fbw-msfs-checklist-fix/src/

!/fbw-msfs-checklist-fix/src/**

/fbw-msfs-checklist-fix/out

Also can't hurt to write a README.md to describe what this folder is for.